vagrant/puppet/cmf/manifests/application/symfony.pp

```diff
@@ -0,0 +1,18 @@
+class cmf::application::symfony(
+  $environment    = hiera('environment', 'dev'),
+  $allow_override = 'None'
+) {
+  validate_string($allow_override)
+
+  $template = $environment ? {
+    'dev'  => 'cmf/vhost_dev.erb',
+    'prod' => 'cmf/vhost_prod.erb',
```

Comment:
instead of two vhost configs we should use one and pass the frontcontroller (app.php or app_dev.php) into the configs